### PR TITLE
Surplus/usd value

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Shows USD value of surplus in confirmation modal and executed order popup if the surplus USD value is larger then 0.01

Confirmation modal
Screenshot 2023-04-19 at 13 42 11

Executed order popup
Screenshot 2023-04-19 at 14 30 24

# To test
1. Create an order and wait until its executed
2. Don't close the confirmation modal
3. Once the order is executed you will see executed order summary after 4 seconds with USD estimation for surplus and also executed order toast popup also with usd estimation for surplus if its larger then 0.01 USD

### CODE REVIEW

File: src/cow-react/common/pure/ExecutedSummary/index.tsx [hunk 1]
```diff
@@ -1,52 +1,35 @@
-import styled from 'styled-components/macro'
-
-import { TokenAmount } from '@cow/common/pure/TokenAmount'
 import { getExecutedSummaryData } from '@cow/utils/getExecutedSummaryData'
 import { Order } from '@src/custom/state/orders/actions'
+import { useCoingeckoUsdValue } from '@src/custom/hooks/useStablecoinPrice'
+import * as styledEl from './styled'
+import { MIN_FIAT_SURPLUS_VALUE } from '@src/custom/constants'
 
-const SummaryWrapper = styled.div`
```
Comment (nenadV91) [inline src/cow-react/common/pure/ExecutedSummary/index.tsx:7]: Moved this to separate styles file

File: src/custom/components/OrderProgressBar/index.tsx [hunk 2]
```diff
@@ -363,84 +359,3 @@ function useGetProgressBarInfo({
     isPending: orderIsPending,
   }
 }
-
```
Comment (nenadV91) [inline src/custom/components/OrderProgressBar/index.tsx:366]: Moved this to also to separate file `src/custom/components/TransactionExecutedContent/styled.ts`

File: src/custom/components/OrderProgressBar/index.tsx [hunk 3]
```diff
@@ -363,84 +359,3 @@ function useGetProgressBarInfo({
...
-    margin-bottom: 5px;
-  }
-`
-
-const Strong = styled.strong`
-  font-size: 0.9rem;
-  white-space: nowrap;
-`
-
-function TransactionExecutedContent({
```
Comment (nenadV91) [inline src/custom/components/OrderProgressBar/index.tsx:403]: Moved this to separate component `src/custom/components/TransactionExecutedContent/index.tsx`

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey Nenad!
Great job!
Can we please add brackets and USD sign for the amount?
brackets and usd
Also, do we need to wrap the value to another line in the pop-up?
another line

Thanks!

**Review by shoom3301 — APPROVED** [review]
Great refactoring! Thank you!

**Comment by nenadV91** [comment]
@elena-zh Should be fixed now.
USD value for surplus where the $ sign is before the value is fine because we are using the same component all over the app

**Comment by alfetopito** [comment]
Hmm the small font looks super odd to me, but I have no better idea.
Can we have @fairlighteth take a look and provide a suggestion?

**Comment by elena-zh** [comment]
Hey @nenadV91 , great job!
Agree with @alfetopito , that a smaller font looks weird. Maybe we can make it's sixe the same as the main text?


Thanks!

**Comment by nenadV91** [comment]
@elena-zh @alfetopito Good suggestion, fixed!

**Comment by nenadV91** [comment]
@elena-zh Ok I think its fixed now 😄

**Review by elena-zh — APPROVED** [review]
Finally, looks good to me!!

**Pull request merged**